spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -458,10 +450,10 @@
           it 'generates a backwards-compatible unique ID' do
             pending_enrollment.update(unique_id: nil)
             stub_request_passed_proofing_results
-            expect(pending_enrollment).to receive(:usps_unique_id).and_call_original
+            expect_any_instance_of(InPersonEnrollment).to receive(:usps_unique_id).and_call_original
```

Comment:
Is there a way we can generate "real" records that fit these various queries?

Reply:
I think that is a good suggestion to go in as a separate ticket. This entire spec file could use a face lift.